src/backend/routes/api.js

```diff
@@ -1,442 +1,562 @@
-module.exports = function(io){
-
-const express = require('express');
-const gameIo = io.of('/game');
-const router = express.Router();
-const queries = require('../db/queries.js');
-
-//retrieve games
-router.get('/games', async (req, res, next) => {
-  try{
-    const games = await queries.getGames();
-    res.send({games: games});
-  }catch(e){
-    res.send({error: e});
-  }
-});
-
-//get game
-router.get('/game/:id', async (req, res, next) => {
-  const id = parseInt(req.params['id'], 10);
-  try{
-    const game = await queries.findGame({gameId: id});
-    res.send({game: game[0]});
-  }catch(e){
-    res.send({error: e});
-  }
-});
+module.exports = function(io) {
+  const express = require("express");
+  const gameIo = io.of("/game");
+  const router = express.Router();
+  const queries = require("../db/queries.js");
+
+  //retrieve games
+  router.get("/games", async (req, res, next) => {
+    try {
+      const games = await queries.getGames();
+      res.send({ games: games });
+    } catch (e) {
+      res.send({ error: e });
+    }
+  });
 
+  //get game
+  router.get("/game/:id", async (req, res, next) => {
```

Comment:
this will match `/game/foobar` and then `parseInt(req.params.id)` will return `NaN`. 

you can restrict this pattern to numeric values with `/game/:id(\\d+)` and then do `const id = Number(req.params.id)`. this applies to all routes with params that are supposed to be numeric

also see the difference between `Number` and `parseInt`
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#Description
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description